README.md

```diff
@@ -1 +1,158 @@
-# obs-opentelemetry
\ No newline at end of file
+# opentelemetry
+
+[Opentelemetry](https://opentelemetry.io/) for [Kitex](https://github.com/cloudwego/kitex)
+
+## Feature
+#### Provider
+- [x] Out-of-the-box  default opentelemetry provider
+
+### Instrumentation
+
+#### Tracing
+- [x] Support server and client kitex rpc tracing
+- [x] Support automatic transparent transmission of peer service through baggage
+
+#### Metrics
+- [x] Support kitex rpc metrics [RED]
+- [x] Support peer service dimension in rpc metrics
+- [x] Support go runtime metrics
+
+#### Logging
+- [x] Extend kitex logger based on logrus
+- [x] Implement tracing auto associated logs
+
+## Server usage
+```go
+import (
+    ...
+    "github.com/kitex-contrib/obs-opentelemetry/provider"
+    "github.com/kitex-contrib/obs-opentelemetry/tracing"
+)
+
+
+func main()  {
+    p := provider.NewOpenTelemetryProvider(
+        provider.WithServiceName("echo"),
+        provider.WithExportEndpoint("localhost:4317"),
+    )
+    defer p.Shutdown(context.Background())
+
+    svr := echo.NewServer(
+        new(EchoImpl),
+        server.WithSuite(tracing.NewServerSuite()),
```

Comment:
is it better to add an option ` server.WithServerBasicInfo(&rpcinfo.EndpointBasicInfo{ServiceName: "echo"})` to specify service name for rpc server? Because the provider set the service name before